### DISCUSSION TITLE
Feature request :  Option to move flag to different namespaces

### DISCUSSION THREAD

**Discussion by achughZscaler** [flipt-io/flipt#1606] [Ideas]
can we have an option to move/Copy  flags between namespaces?

**Comment by markphelps** [comment]
Yup great idea. Could likely implement something in the ui to copy flags and segments. Rules might be a little trickier though. But as a first pass I think flags w variants and segments w constraints should be doable

**Reply by markphelps** [reply]
Hey @achughZscaler this was just added in the latest version. Lmk what you think!
